core/src/main/resources/hudson/security/GlobalSecurityConfiguration/help-markupFormatter.html

```diff
@@ -0,0 +1,24 @@
+<div>
+  <p>
+    Jenkins shows user-provided descriptions on objects like jobs, views, or builds.
+    The markup formatter controls how those descriptions are rendered.
+  </p>
+  <p>
+    &ldquo;Plain text&rdquo; is the default formatter and renders the provided description largely as entered.
+  </p>
+  <p>
+    Plugin may contribute additional markup formatters that support HTML or other markup languages.
```

Comment:
I think this should be plural.  I can install the Markdown formatter plugin and the OWASP Markup Formatter on a Jenkins controller and switch between them.
```suggestion
    Plugins may contribute additional markup formatters that support HTML or other markup languages.
```